common/utils/src/main/resources/error/error-classes.json

```diff
@@ -463,6 +463,12 @@
     ],
     "sqlState" : "42704"
   },
+  "COLLATION_SUPPORT_DISABLED" : {
+    "message" : [
+      "Collation feature is under development and not all features are supported. To enable existing features toggle `COLLATION_ENABLED` flag to true."
```

Comment:
```suggestion
      "Collation support is under development and not all features are supported. To enable existing features set `spark.sql.collation.enabled` to `true`."
```

Reply:
Out of curiosity: do we have or want any mechanism to disallow the enablement of the feature?

Reply:
If I understood correctly, we want to make sure that customer is prevented from trying to use partially built feature and getting in trouble because some functionality was not implemented at the time.

Reply:
This is a good question. I am not sure what is the policy here - @cloud-fan and @MaxGekk - can you guide us here?

Reply:
Preventing the use of a partially baked feature is a good reason here.